strongbox-storage/strongbox-storage-core/src/main/java/org/carlspring/strongbox/io/ArtifactInputStream.java

```diff
@@ -40,17 +49,31 @@
 
     private Map<String, String> hexDigests = new LinkedHashMap<>();
 
+    private String extension;
+
     public ArtifactInputStream(ArtifactCoordinates coordinates,
                                InputStream is,
                                Set<String> checkSumDigestAlgorithmSet)
         throws NoSuchAlgorithmException
     {
         super(is);
         this.artifactCoordinates = coordinates;
+
         for (String algorithm : checkSumDigestAlgorithmSet)
         {
             addAlgorithm(algorithm);
         }
+
+    }
+
+    private boolean isInputStreamValid()
+            throws IOException
+    {
+        if (in.available()>0)
```

Comment:
Please, always use braces, even for one-liners. Could you please change this across your pull?

Reply:
Fixed.